drivers/SmartThings/zigbee-thermostat/src/stelpro/stelpro_maestrostat.lua

```diff
@@ -21,7 +21,7 @@ local Thermostat = clusters.Thermostat
 local ThermostatUserInterfaceConfiguration = clusters.ThermostatUserInterfaceConfiguration
 
 local STELPRO_THERMOSTAT_FINGERPRINTS = {
-  { mfr = "Stelpro", model = "MaestroStat" },
+  { mfr = "Stello", model = "HT402" },
```

Comment:
Please omit this change from the PR as we want to limit this PR to be adding a new device that is pursuing WWST. Thank you.